content/en/docs/concepts/architecture/nodes.md

```diff
@@ -513,6 +513,29 @@ During a non-graceful shutdown, Pods are terminated in the two phases:
   recovered since the user was the one who originally added the taint.
 {{< /note >}}
 
+### Disable force detach on timeout {#disable-force-detach-on-timeout}
+
+With the advent of the "Non-graceful node shutdown" feature, users might opt
+to disable force detach on timeout in
+[kube controller manager](/docs/reference/command-line-tools-reference/kube-controller-manager/).
+
```

Comment:
Kind of...

I think the point I'm trying to make is that k8s can force detach an existing PersistentVolume after the control plane _thinks_ that a node has failed (i.e., is "Unhealthy").

However, in a number of our testcases this behavior leads to data corruption. Different folks might have different workflows, so this might not necessarily apply to everyone. But we need a way to shut off this timer.

In general, this is a common refrain... It feels similar in nature to not being able to [stop threads in Java](https://wiki.sei.cmu.edu/confluence/display/java/THI05-J.+Do+not+use+Thread.stop()+to+terminate+threads).

If the timer is disabled, then pods which misbehave (and ignore signals to shut themselves down) on "Unhealthy" nodes will stick around until the user uses the "Non-graceful node shutdown" feature to take the node out-of-commission.

Reply:
Maybe a brief restatement of the underlying problem that will help people with no familiarity grasp what both of these features are trying to solve and why the new way is better. If you want me to take a swing at the wording I can.

Reply:
Thanks @bswartz! I'd certainly appreciate the help.

I'll throw something on your calendar for later in the week; we can chat some more.